dashboards/Node_Temperature_Details.json

```diff
@@ -282,7 +282,7 @@
             "dashes": false,
             "datasource": "Prometheus",
             "decimals": 0,
-            "description": "Hardware monitoring devices let you monitor the hardware health of a system. Most modern motherboards include such a device. It can include temperature sensors, voltage sensors, fan speed sensors and various additional features such as the ability to control the speed of the fans. ",
+            "description": "Hardware monitoring devices let you monitor the hardware health of a system. Most modern motherboards include such a device. It can include temperature sensors, voltage sensors, fan speed sensors and various additional features such as the ability to control the speed of the fans. \n\nIf shown temperature is 511C it means that reading of a chip sensor is failed. Please consider to install/update a driver and check if sensors are properly configured.",
```

Comment:
I'd suggest this version:
`If the temperature is 511C, it means that reading of a chip sensor has failed. Please consider installing/updating a driver and check if the sensors are properly configured.`